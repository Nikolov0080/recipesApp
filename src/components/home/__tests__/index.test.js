import React from 'react'
import ReactDOM from 'react-dom';
import Home from '../index';
import {  cleanup } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import renderer from 'react-test-renderer';

afterEach(cleanup);

it('renders Home OK', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Home />, div);
    ReactDOM.unmountComponentAtNode(div);
});

it('matches snapshot', () => {
    const three = renderer.create(<Home/>).toJSON();
    expect(three).toMatchSnapshot()
});