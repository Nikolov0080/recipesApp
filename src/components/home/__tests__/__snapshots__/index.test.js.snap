// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div>
  <div
    className="big_box"
  >
    <div
      className="container row"
    >
      <div
        className="left_col col"
      >
        Click on the image to view the recipes in the category
      </div>
      <div
        className="col-lg-6 col-sm"
      >
        <div>
          <div
            className="main_box carousel slide"
            onKeyDown={[Function]}
            onMouseOut={[Function]}
            onMouseOver={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
          >
            <ol
              className="carousel-indicators"
            >
              <li
                className="active"
                onClick={[Function]}
              />
              <li
                onClick={[Function]}
              />
              <li
                onClick={[Function]}
              />
              <li
                onClick={[Function]}
              />
              <li
                onClick={[Function]}
              />
              <li
                onClick={[Function]}
              />
              <li
                onClick={[Function]}
              />
              <li
                onClick={[Function]}
              />
              <li
                onClick={[Function]}
              />
              <li
                onClick={[Function]}
              />
              <li
                onClick={[Function]}
              />
              <li
                onClick={[Function]}
              />
            </ol>
            <div
              className="carousel-inner"
            >
              <div
                className="current active carousel-item"
              >
                <img
                  alt="view"
                  className="cat_img"
                  onClick={[Function]}
                  src="https://res.cloudinary.com/recepsbg/image/upload/v1608240048/Categories/Lunch_hqv0yt.png"
                />
                <div
                  className="carousel-caption"
                >
                  <h3
                    className="cat_name"
                  >
                    Lunch
                  </h3>
                </div>
              </div>
              <div
                className="current carousel-item"
              >
                <img
                  alt="view"
                  className="cat_img"
                  onClick={[Function]}
                  src="https://res.cloudinary.com/recepsbg/image/upload/v1608240048/Categories/Soups_epim7e.png"
                />
                <div
                  className="carousel-caption"
                >
                  <h3
                    className="cat_name"
                  >
                    Soups
                  </h3>
                </div>
              </div>
              <div
                className="current carousel-item"
              >
                <img
                  alt="view"
                  className="cat_img"
                  onClick={[Function]}
                  src="https://res.cloudinary.com/recepsbg/image/upload/v1608240048/Categories/Bread_ezhdwg.png"
                />
                <div
                  className="carousel-caption"
                >
                  <h3
                    className="cat_name"
                  >
                    Bread
                  </h3>
                </div>
              </div>
              <div
                className="current carousel-item"
              >
                <img
                  alt="view"
                  className="cat_img"
                  onClick={[Function]}
                  src="https://res.cloudinary.com/recepsbg/image/upload/v1608240049/Categories/breakfast_zgx3oz.png"
                />
                <div
                  className="carousel-caption"
                >
                  <h3
                    className="cat_name"
                  >
                    Breakfast
                  </h3>
                </div>
              </div>
              <div
                className="current carousel-item"
              >
                <img
                  alt="view"
                  className="cat_img"
                  onClick={[Function]}
                  src="https://res.cloudinary.com/recepsbg/image/upload/v1608240048/Categories/Salads_vr4xbz.png"
                />
                <div
                  className="carousel-caption"
                >
                  <h3
                    className="cat_name"
                  >
                    Salads
                  </h3>
                </div>
              </div>
              <div
                className="current carousel-item"
              >
                <img
                  alt="view"
                  className="cat_img"
                  onClick={[Function]}
                  src="https://res.cloudinary.com/recepsbg/image/upload/v1608242232/Categories/dinner_bkaamm.png"
                />
                <div
                  className="carousel-caption"
                >
                  <h3
                    className="cat_name"
                  >
                    Dinners
                  </h3>
                </div>
              </div>
              <div
                className="current carousel-item"
              >
                <img
                  alt="view"
                  className="cat_img"
                  onClick={[Function]}
                  src="https://res.cloudinary.com/recepsbg/image/upload/v1608240049/Categories/Drinks_qn658g.png"
                />
                <div
                  className="carousel-caption"
                >
                  <h3
                    className="cat_name"
                  >
                    Drinks
                  </h3>
                </div>
              </div>
              <div
                className="current carousel-item"
              >
                <img
                  alt="view"
                  className="cat_img"
                  onClick={[Function]}
                  src="https://res.cloudinary.com/recepsbg/image/upload/v1608240048/Categories/Pasta_qay9ds.png"
                />
                <div
                  className="carousel-caption"
                >
                  <h3
                    className="cat_name"
                  >
                    Pasta
                  </h3>
                </div>
              </div>
              <div
                className="current carousel-item"
              >
                <img
                  alt="view"
                  className="cat_img"
                  onClick={[Function]}
                  src="https://res.cloudinary.com/recepsbg/image/upload/v1608240048/Categories/sandwiches_b3dgbu.png"
                />
                <div
                  className="carousel-caption"
                >
                  <h3
                    className="cat_name"
                  >
                    Sandwiches
                  </h3>
                </div>
              </div>
              <div
                className="current carousel-item"
              >
                <img
                  alt="view"
                  className="cat_img"
                  onClick={[Function]}
                  src="https://res.cloudinary.com/recepsbg/image/upload/v1608240049/Categories/american_cwvrgx.png"
                />
                <div
                  className="carousel-caption"
                >
                  <h3
                    className="cat_name"
                  >
                    American
                  </h3>
                </div>
              </div>
              <div
                className="current carousel-item"
              >
                <img
                  alt="view"
                  className="cat_img"
                  onClick={[Function]}
                  src="https://res.cloudinary.com/recepsbg/image/upload/v1608240048/Categories/Sweets_rez2ox.png"
                />
                <div
                  className="carousel-caption"
                >
                  <h3
                    className="cat_name"
                  >
                    Sweets
                  </h3>
                </div>
              </div>
              <div
                className="current carousel-item"
              >
                <img
                  alt="view"
                  className="cat_img"
                  onClick={[Function]}
                  src="https://res.cloudinary.com/recepsbg/image/upload/v1608240049/Categories/diet_pq9e0b.png"
                />
                <div
                  className="carousel-caption"
                >
                  <h3
                    className="cat_name"
                  >
                    Low calories
                  </h3>
                </div>
              </div>
            </div>
            <a
              className="carousel-control-prev"
              href="#"
              onClick={[Function]}
              onKeyDown={[Function]}
              role="button"
            >
              <span
                aria-hidden="true"
                className="carousel-control-prev-icon"
              />
              <span
                className="sr-only"
              >
                Previous
              </span>
            </a>
            <a
              className="carousel-control-next"
              href="#"
              onClick={[Function]}
              onKeyDown={[Function]}
              role="button"
            >
              <span
                aria-hidden="true"
                className="carousel-control-next-icon"
              />
              <span
                className="sr-only"
              >
                Next
              </span>
            </a>
          </div>
        </div>
      </div>
      <div
        className="right_col col"
      >
        <div>
          <div
            className="card card"
          >
            <div
              className="card-title h5"
            >
               
              <h2>
                Random facts
              </h2>
            </div>
            <p
              className="fact_text card-text"
            >
              
            </p>
            <button
              className="mt-auto btn btn-outline-primary"
              disabled={false}
              onClick={[Function]}
              type="button"
            >
              Next
            </button>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    className="loading_container"
  >
    <span
      className="sc-AxjAm jfcyvh"
      data-testid="skeleton__item"
      height="1000px"
      width="100%"
    >
      <h1
        className="pulsate"
      >
        Loading...
      </h1>
    </span>
  </div>
</div>
`;
